external_search.py

```diff
@@ -49,6 +49,15 @@ def __init__(self, url=None, works_index=None):
         self.exists = self.__client.exists
 
     def setup_index(self):
+        """
+        Create the search index with appropriate mapping.
+
+        This will destroy the search index, and all works will need
+        to be indexed again. In production, don't use this on an
+        existing index. Use it to create a new index, then change the 
+        alias to point to the new index.
+        """
+
```

Comment:
How do you use this to create a new index if it always operates on self.works_index? That seems unreliable. Would it make sense to have a method that orchestrates the whole create-a-new-index and change-the-alias process?